Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabStrip.css

```diff
@@ -84,3 +90,9 @@
   border-color: var(--color-disabled-border);
   background-color: var(--color-disabled-background);
 }
+
+.tabStripTabWrapper .tabStripTab a span.anchorBody,
+.tabStripTabWrapper .tabStripTabSelected a span.anchorBody
+{
+  display: flex;
```

Comment:
Why do we not need `gap` here, but we do in webbutton?

Reply:
Because of the way the tab strip is rendered... It renders a NBSP and a span containing the text, which is interpreted by flexbox as two separate elements and as such the NBSP is acting as a spacer. In the WebButton there is no span being rendered. The NBSP and the text is treated as the same element and as such we need a gap to have spacing between the icon and the text.

Reply:
hmm...since the spacing is probably not identical, maybe we could easily select and then set the nbsp to hidden via css and then use gap for the rest. don't know, didn't try. or, perhaps somehting for a follow-up refacotring task to clean this up.